tests.gradle

```diff
@@ -55,12 +57,11 @@ if(file('modules').exists() && file('modules').isDirectory()){
 if(file('modules_core').exists() && file('modules_core').isDirectory()){
     file('modules_core').eachDir {
         sourceSets.test.java.srcDirs += it.toString()+"/src-test"
+        sourceSets.test.resources.srcDirs +=  it.toString()+"/src-test/resources"
```

Comment:
Is this format ok? If not, could we have the sum sign with spaces before and after it? Sorry for the annoyance :sweat_smile: